src/coreclr/jit/lclvars.cpp

```diff
@@ -4572,7 +4581,9 @@ void Compiler::lvaComputeRefCounts(bool isRecompute, bool setSlotNumbers)
     {
         if (block->IsLIR())
         {
+#if !defined(TARGET_WASM) // TODO: maybe we should be setting isRecompute to true?
```

Comment:
Not necessarily (it would be a "lie" if we didn't run any refcounts before). I think it is fine to leave the `#ifdef` (or, to be honest, it should be fine to delete the assert altogether; it seems low value).
```suggestion
#if !defined(TARGET_WASM)
```